### PR TITLE
[Excel] (Custom functions) Updates to requirements note, adds volatile sample

### PR DESCRIPTION
**Pull request by mscharlock**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[7c4a890](https://github.com/OfficeDev/office-js-docs-pr/pull/1094/commits/7c4a8909dc989c6b1e14417d4231d63602777bef)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-authentication.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-authentication.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-batching.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-batching.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-debugging.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-debugging.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-dialog.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-dialog.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-errors.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-errors.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-json-autogeneration.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-localize.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-localize.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-naming.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-naming.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-parameter-options.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-parameter-options.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-requirements.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-runtime.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-save-state.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-save-state.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-volatile.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-volatile.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-web-reqs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-web-reqs.md) | :white_check_mark:Succeeded | |
[docs/excel/make-custom-functions-compatible-with-xll-udf.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/make-custom-functions-compatible-with-xll-udf.md) | :white_check_mark:Succeeded | |
[docs/excel/xll-compatibility-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/xll-compatibility-requirements.md) | :white_check_mark:Succeeded | |
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/toc.yml) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906171904001086-1094%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=A1wR%2FF8%2B50NvqVZ5mOEOvDW1IzY1MRoUqVRoSdCxJoE%3D&st=2019-06-17T19%3A00%3A12Z&se=2019-07-18T19%3A05%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[1744423](https://github.com/OfficeDev/office-js-docs-pr/pull/1094/commits/17444231d056e3f5d6d44074171e789bfa817984)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-authentication.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-authentication.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-batching.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-batching.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-debugging.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-debugging.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-dialog.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-dialog.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-errors.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-errors.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-json-autogeneration.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-localize.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-localize.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-naming.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-naming.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-parameter-options.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-parameter-options.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-requirements.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-runtime.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-save-state.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-save-state.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-volatile.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-volatile.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-web-reqs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-web-reqs.md) | :white_check_mark:Succeeded | |
[docs/excel/make-custom-functions-compatible-with-xll-udf.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/make-custom-functions-compatible-with-xll-udf.md) | :white_check_mark:Succeeded | |
[docs/excel/xll-compatibility-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/xll-compatibility-requirements.md) | :white_check_mark:Succeeded | |
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/toc.yml) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906180005319746-1094%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=A7QUrFISigiYZhLZjlUOohCm2Xjibq2QYNdrI4cNchI%3D&st=2019-06-18T00%3A01%3A41Z&se=2019-07-19T00%3A06%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[a57cd9f](https://github.com/OfficeDev/office-js-docs-pr/pull/1094/commits/a57cd9fc84c0c280484b1b81701db130cd713d0e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-authentication.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-authentication.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-batching.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-batching.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-debugging.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-debugging.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-dialog.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-dialog.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-errors.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-errors.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-json-autogeneration.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-localize.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-localize.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-naming.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-naming.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-parameter-options.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-parameter-options.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-requirements.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-runtime.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-save-state.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-save-state.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-volatile.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-volatile.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-web-reqs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-web-reqs.md) | :white_check_mark:Succeeded | |
[docs/excel/make-custom-functions-compatible-with-xll-udf.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/make-custom-functions-compatible-with-xll-udf.md) | :white_check_mark:Succeeded | |
[docs/excel/xll-compatibility-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/xll-compatibility-requirements.md) | :white_check_mark:Succeeded | |
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/toc.yml) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906180006282963-1094%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=s%2FKAzujENoAVfbtQDfqZQ1GV7mCureIJDCdJZvM90mM%3D&st=2019-06-18T00%3A02%3A52Z&se=2019-07-19T00%3A07%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[2fd4902](https://github.com/OfficeDev/office-js-docs-pr/pull/1094/commits/2fd490259f48b78b316ed898b887df6b54cc197f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-authentication.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-authentication.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-batching.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-batching.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-debugging.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-debugging.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-dialog.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-dialog.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-errors.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-errors.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-json-autogeneration.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-localize.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-localize.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-naming.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-naming.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-parameter-options.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-parameter-options.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-requirements.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-runtime.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-save-state.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-save-state.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-volatile.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-volatile.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-web-reqs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-web-reqs.md) | :white_check_mark:Succeeded | |
[docs/excel/make-custom-functions-compatible-with-xll-udf.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/make-custom-functions-compatible-with-xll-udf.md) | :white_check_mark:Succeeded | |
[docs/excel/xll-compatibility-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/xll-compatibility-requirements.md) | :white_check_mark:Succeeded | |
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/toc.yml) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906180007165305-1094%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=sOumcq%2BxaSJ%2BaH63qD1EpY%2Bp9drALYcbGtdkvlBLhJs%3D&st=2019-06-18T00%3A04%3A07Z&se=2019-07-19T00%3A09%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-overview.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-ms.date: 05/15/2019
+ms.date: 06/17/2019
```
[the remark this answers is not in the document]
Comment (mscharlock) [reply docs/excel/custom-functions-overview.md]: Sorry - I think I changed them even though I decided not to remove the note in some of the them. I'll revert them! :) Thanks

File: docs/excel/custom-functions-authentication.md [hunk 2]
```diff
@@ -9,8 +9,6 @@ localization_priority: Priority
 
 In some scenarios your custom function will need to authenticate the user in order to access protected resources. While custom functions don't require a specific method of authentication, you should be aware that custom functions run in a separate runtime from the task pane and other UI elements of your add-in. Because of this, you'll need to pass data back and forth between the two runtimes using the `OfficeRuntime.storage` object and the Dialog API.
 
-[!include[Excel custom functions note](../includes/excel-custom-functions-note.md)]
```
[the remark this answers is not in the document]
Comment (mscharlock) [reply docs/excel/custom-functions-authentication.md]: Yep, still being used in a number of places :)

**Comment by o365devx** [comment]
Docs Build status updates of commit _[0105053](https://github.com/OfficeDev/office-js-docs-pr/pull/1094/commits/01050533b43ed24d710b519013c58bbe4d4f1135)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-authentication.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-authentication.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-batching.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-batching.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-debugging.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-debugging.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-dialog.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-dialog.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-errors.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-errors.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-json-autogeneration.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-localize.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-localize.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-naming.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-naming.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-parameter-options.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-parameter-options.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-requirements.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-save-state.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-save-state.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-volatile.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/custom-functions-volatile.md) | :white_check_mark:Succeeded | |
[docs/excel/xll-compatibility-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/excel/xll-compatibility-requirements.md) | :white_check_mark:Succeeded | |
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/toc.yml) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-requirements/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906180012191487-1094%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2FtJS%2F4M37BDeJt7hpODsP6ftuZ1RJtewrlBK5mVyxfU%3D&st=2019-06-18T00%3A08%3A27Z&se=2019-07-19T00%3A13%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**